### PR TITLE
AudioSet20k datamodule + model refactoring

### PR DESCRIPTION
**Pull request by The-Mats**
In this branch we integrate AS-20k into the BS Pipeline to test if our models run as expected. For this we use the huggingface dataset: https://huggingface.co/datasets/confit/audioset-20k-wds

For this we need to do the following:

- [x] Create datamodule
- [x]  Create experiments
- [ ]  Check EAT-SSL
- [x]  Check BEATs
- [x]  Check AudioMAE -> Way better performance

At the moment there is an error which can be fixed when setting `num_proc` to 1 in the base datamodule where it is hard coded, so we have to integrate this into the Audioset Datamodule

@tom2208 @raphaelschwinger I couldn't get any decent accuracies for now when not training at all for BEATs and EAT-SSL:
### BEATs

The correct settings and checkpoint are provided. 
The output directly is a probability which is why the base module need to get some kind of identity function as `output_activation`. But even if I comment softmax out the probabilities have a really low accuracy

### EAT-SSL

Started looking into the model but it is quiet complicated and I couldn't find an easy way to extract logits

## Offtopic
- Refactor `BirdSetModule` to `BioFoundationModule`
- Refactored AudioMAE preprocessing to use torchaudio
- Fixed preprocessing of ConvNext_BS
- Surpress tensorflow warnings in `train.sh`

### CODE REVIEW

**Comment by raphaelschwinger** [comment]
@XgamerTV @tom2208 I am taking a look into this now :)
